src/builder/timer.rs

```diff
@@ -3,42 +3,39 @@ use std::io::Write;
 use std::path::Path;
 use std::fs::File;
 use std::fmt::{Arguments};
-
-use time::{self, Timespec};
+use std::time::{Instant, Duration};
 
 const NANO_FACTOR: f64 = 0.000000001;
 
 
-fn time_as_f64(tm: Timespec) -> f64 {
-    (tm.sec as f64) + (NANO_FACTOR * tm.nsec as f64)
+fn duration_as_f64(d: Duration) -> f64 {
+    (d.as_secs() as f64) + (NANO_FACTOR * d.subsec_nanos() as f64)
 }
 
 pub struct TimeLog {
     file: File,
-    start: Timespec,
-    prev: Timespec,
+    start: Instant,
+    prev: Instant,
 }
 
-
 impl TimeLog {
     pub fn start(path: &Path) -> Result<TimeLog, Error> {
-        let tm = time::get_time();
+        let now = Instant::now();
         let mut res = TimeLog {
             file: try!(File::create(path)),
-            start: tm,
-            prev: tm,
+            start: now,
+            prev: now,
         };
-        try!(res.mark(format_args!("Start {}", time_as_f64(tm))));
+        try!(res.mark(format_args!("Start {:?}", now)));
```

Comment:
Not sure about this one. It used to be a unix timestamp. Given that it's easy to convert instant to a unixtime, i think we should keep it that way.